Week-4/guessingGame/script.js

```diff
@@ -0,0 +1,38 @@
+function playGuessingGame(numToGuess, totalGuesses = 10) {
+    let guessesTaken = 0;
+    let userGuess = 0;
+    let promptMessage = "Enter a number between 1 and 100.";
+
+    while (guessesTaken < totalGuesses) {
```

Comment:
Consider adding a feature to quit the game if the user enters a specific word, like 'quit', to provide an escape hatch from the while loop.